src/PartialLoopPurityPass.cpp

```diff
@@ -19,25 +19,32 @@
 
 #include <config.h>
 
-#include "CheckModule.h"
 #include "PartialLoopPurityPass.h"
-#include "SpinAssumePass.h"
-#include "Debug.h"
+
 #include "vecset.h"
+
```

Comment:
Why do we put a space here? `vecset.h` is a nidhugg header like the others below. If the different capitalisation is an issue, we should fix that instead.

Reply:
Corrected.